app/javascript/components/deployedVersionCompare.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Subscription } from 'react-apollo';
+import gql from 'graphql-tag';
+
+const getVersion = gql`
+  subscription onDeploymentSha {
+    deploymentSha
+  }
+`;
+
+let lastValue = null;
+
+export const Version = () => (
+  <Subscription subscription={getVersion}>
+    {({ loading, error, data }) => {
+      if (loading || error) {
+        return null;
+      }
+
+      const newValue = data.deploymentSha;
+      if (lastValue == null) {
+        lastValue = newValue;
+      } else if (newValue !== lastValue) {
+        lastValue = newValue;
+        window.location.reload();
+      }
+
+      return newValue;
```

Comment:
This will render some content into the DOM. I don't think we need to render anything at all. In that case, could we skip using React entirely by calling the apollo client API directly with a subscription?